sdk/core/System.ClientModel/src/Options/ClientPipelineOptions.cs

```diff
@@ -67,6 +89,20 @@ public class ClientPipelineOptions
 
     #region Pipeline creation: Policy settings
 
+    /// <summary>
+    /// TODO
+    /// </summary>
+    public LoggingOptions? LoggingOptions
```

Comment:
Wondering if this should be non-nullable and we always instantiate, like in Azure.Core's ClientOptions?  That would give us a good place to store defaults.